frontend/src/hooks/useActions.ts

```diff
@@ -590,4 +590,8 @@ const promptLoadFile = (parse, onData, errorMessage = 'Failed to parse file') =>
   input.click()
 }
 
+function showWarning (msg: string) {
+  dispatchCustomEvent('showWarning', msg)
+}
+
```

Comment:
Useful function
Put it inside the warning file and just import it in other places

Maybe a cheeky little bit of documentation saying it shows the modal 👀

Reply:
Good suggestion, all done